### ISSUE TITLE
Sucursales en ticket del pos

### ISSUE THREAD

**Issue by celm1990** [dansanti/l10n_cl_dte_point_of_sale#19]
En un ambiente multi tiendas, y revisando el formato de factura para el pos que pone como ejemplo el sii, hay un campo para Matriz y un campo para Sucursal, pero en la configuracion del pos y en el ticket que hay actualmente, no veo x ningun lado que se pueda configurar eso, me podrias indicar si hay soporte para eso, o como lo podriamos solucionar(se me ocurre agregar un campo m2o hacia res.partner para seleccionar la direccion de la sucursal en cada TPV, la direccion matriz si se toma de la compañia)
![image](https://user-images.githubusercontent.com/7775116/35603605-14393216-060c-11e8-9138-613336f575a8.png)

**Comment by dansanti** [comment]
en el anterior issue te referías a este formato de factura?, no le he dado soporte aún a este tipo. El asunto de multicompañia en odoo 9 no funcionaba bien del todo así que dejé de lado eso, creo aquí en 11 funciona relativamente bien, pero no he tenido tiempo, hay otras cosas más importantes que estoy viendo ahora

**Comment by dansanti** [comment]
además de que meterse en factura desde pos, hay que agregar varias cosas, como lo son las referencias, descuentos globales, etc

**Comment by celm1990** [comment]
Tome la muestra del sii que es de Factura en formato ticket, pero mas bien era orientado al formato de la boleta del pos, no tanto de la factura. En el cual tambien va el tema de la sucursal, y no estoy hablando de multi compañia, sino multi tiendas en el contexto de un punto de venta. No veo por ningun lado donde pueda configurar la sucursal en el TPV y eso tomarlo para imprimir en el ticket de boleta.

Procedo a agregar  el campo m2o como mensiono arriba???

**Comment by celm1990** [comment]
Este es un ejemplo del ticket de V10 que nosotros modificamos
![image](https://user-images.githubusercontent.com/7775116/35604459-4eea2a42-0610-11e8-8241-c031d027e1f4.png)
Si vez Intersport tiene varias tiendas en todo el pais, y en la parte de sucursal indicamos eso.

Espero que sea claro lo que estoy preguntando

**Comment by dansanti** [comment]
NO, porque la idea es que lo tome desde el diario, tendrías que acceder a sucursal_id del diario, todo se define en el diaro para las facuras, para las guía se define en la localidad(no recuerdo si ya agregué esa parte)

**Comment by celm1990** [comment]
Pero eso implicaria crear un diario por cada tienda, en un principio habiamos hecho asi, un diario por cada tienda pero fue un desastre eso, al final tuvimos que cambiar a un solo diario de factura y en este diario agregar los tipos de documento(1 tipo de boleta x cada tienda con una secuencia y CAF diferente) y en el TPV escogiamos ese tipo de documento especifico para la tienda.
